shared/actions/platform-specific/index.desktop.js

```diff
@@ -0,0 +1,229 @@
+// @flow
+import * as ConfigGen from '../config-gen'
```

Comment:
moves of this logic moved from ../platform-specific.desktop.js. also moved some things into here (like pipe ownership check) and removed native only stuff